xcconfig/base.xcconfig

```diff
@@ -0,0 +1,43 @@
+SDKROOT = iphoneos
+ONLY_ACTIVE_ARCH = YES
+DEFINES_MODULE = YES
+PRODUCT_NAME = $(TARGET_NAME:c99extidentifier)
+
+CURRENT_PROJECT_VERSION = 1
+VERSION_INFO_PREFIX =
+VERSIONING_SYSTEM = apple-generic
+TARGETED_DEVICE_FAMILY = 1,2
+IPHONEOS_DEPLOYMENT_TARGET = 11.3
+DYLIB_COMPATIBILITY_VERSION = 1
+DYLIB_CURRENT_VERSION = 1
+DYLIB_INSTALL_NAME_BASE = @rpath
+LD_RUNPATH_SEARCH_PATHS = @executable_path/Frameworks @loader_path/Frameworks
+
+SWIFT_VERSION = 4.0
+ENABLE_BITCODE = NO
+
+SWIFT_OPTIMIZATION_LEVEL_debug = -Onone
+SWIFT_OPTIMIZATION_LEVEL_release = -O
+SWIFT_OPTIMIZATION_LEVEL = $(SWIFT_OPTIMIZATION_LEVEL_$(buildvariant))
+
+SWIFT_ACTIVE_COMPILATION_CONDITIONS_debug = DEBUG
+SWIFT_ACTIVE_COMPILATION_CONDITIONS_release =
+SWIFT_ACTIVE_COMPILATION_CONDITIONS = $(SWIFT_ACTIVE_COMPILATION_CONDITIONS_$(buildvariant))
+
+GCC_OPTIMIZATION_LEVEL_debug = 0
+GCC_OPTIMIZATION_LEVEL_release = s
+GCC_OPTIMIZATION_LEVEL = $(GCC_OPTIMIZATION_LEVEL_$(buildvariant))
+
+GCC_PREPROCESSOR_DEFINITIONS_debug = DEBUG=1
+GCC_PREPROCESSOR_DEFINITIONS_release =
+GCC_PREPROCESSOR_DEFINITIONS = $(GCC_PREPROCESSOR_DEFINITIONS_$(buildvariant))
+
+GCC_DYNAMIC_NO_PIC = NO
+GCC_NO_COMMON_BLOCKS = YES
+GCC_C_LANGUAGE_STANDARD = gnu11
+CLANG_CXX_LANGUAGE_STANDARD = gnu++14
+CLANG_CXX_LIBRARY = libc++
+CLANG_ENABLE_MODULES = YES
+CLANG_ENABLE_OBJC_ARC = YES
```

Comment:
I am not sure why there are OBJC settings like this specified (there is no objective-c being used that I am aware of), please double check this is valid.